src/lib/seam/client.ts

```diff
@@ -1,18 +1,17 @@
 import { useQuery } from '@tanstack/react-query'
-import { useContext } from 'react'
 import { type ClientSession, Seam } from 'seamapi'
 import { v4 as uuidv4 } from 'uuid'
 
-import { seamContext } from 'lib/provider.js'
+import { useSeamComponents } from 'lib/SeamComponents.js'
 
-export function useSeam(): {
+export function useSeamClient(): {
```

Comment:
I left this as `useSeam` intentionally (considered `useSeamClient`) but I wanted to leave the option open to put more in the context. Not I'm not so sure and think this change is probably correct!

Reply:
This is specifically for the API client, though. We still have `useSeamContext` in `lib/SeamProvider`